server/src/services/server.service.ts

```diff
@@ -131,11 +132,26 @@ export class ServerService extends BaseService {
       usage.usageVideos = user.usageVideos;
       usage.quotaSizeInBytes = user.quotaSizeInBytes;
 
+      for (const path of user.encodedVideoPaths) {
+        if (path.trim() === '') {
+          continue;
+        }
+
+        try {
+          const stats = fs.statSync(path);
```

Comment:
Pretty sure we already have a repo method for that. If we don't, we should add one